database/ffldb/transaction_test.go

```diff
@@ -0,0 +1,564 @@
+package ffldb
+
+import (
+	"bytes"
+	"github.com/kaspanet/kaspad/database"
+	"io/ioutil"
+	"strings"
+	"testing"
+)
+
+func TestKeyValueTransactionCommit(t *testing.T) {
```

Comment:
Can you think of a better name?

Reply:
And if not - add comment explaining test